changelog/unreleased/kong/cluster_events_invalidation_channel_deprecation.yml

```diff
@@ -0,0 +1,3 @@
+message: Starting with this version, the "invalidation" cluster event channel is marked as deprecated and is slated for removal in upcoming releases. Developers are advised to remove any relevant subscribers to this channel.
```

Comment:
What is the rationale of deprecating the old channel? Leaving it as is for the db cache, plus the separate one for the core cache, should be enough for now, and even for future improvements.

Reply:
Removed for now. My original intent was to let every cache system entry have its own cluster event channel with a clear name, instead of channels like "invalidation"(that might be reused when seen by developers), since the PR originally wants to reduce useless worker events being generated due to cache invalidation

Reply:
IMO what we have is enough; the future renaming would cause us more harm than good I'd say.